pkg/logging/logging.go

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package logging
+
+import "log"
+
+func Fatal(str string) {
+	log.Fatal(str)
+}
```

Comment:
I'm not sure I understand the purpose of this package

Reply:
Both code lines:
- `log.Fatal(fmt.Sprintf(ACLTypeNotSupported, string(conn.Src.Type)))`
- `log.Fatalf(fmt.Sprintf(ACLTypeNotSupported, string(conn.Src.Type)))`

raise a linter error:
- `S1038: should use log.Fatalf(...) instead of log.Fatal(fmt.Sprintf(...)) (gosimple)`
- `printf: non-constant format string in call to log.Fatalf (govet)`